src/styles/components/_product-information.scss

```diff
@@ -1,20 +1,122 @@
 .product-information {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
   box-sizing: border-box;
-  padding: rem(16px);
+  padding: rem(18px 24px);
+
+  @include respond-above('md') {
+    padding: 0;
+  }
+
+  &__header,
+  &__body {
+    @include respond-above('md') {
+      background-color: $alabaster;
+    }
+  }
 
   &__header {
-    margin-bottom: rem(24px);
+    display: flex;
+    align-items: center;
+
+    @include respond-above('md') {
+      width: rem(200px);
+      padding: rem(20px) 0 0 rem(28px);
+      border-top-left-radius: rem(32px);
+    }
+
+    @include respond-above('lg') {
+      width: rem(220px);
+      padding: rem(24px) 0 0 rem(32px);
+    }
+  }
+
+  &__subheader {
+    display: none;
```

Comment:
Quick question, so this is intentionally hidden?

Reply:
Yep there is no `subheader` in mobile. References are the screenshots in PR description.